autokeras/blocks/basic.py

```diff
@@ -260,6 +261,222 @@ def _get_padding(kernel_size, output_node):
         return 'same'
 
 
+class MultiHeadSelfAttention(block_module.Block):
```

Comment:
Please make sure to follow the API for the official MultiHeadAttention layer. https://github.com/tensorflow/community/pull/260

Reply:
Currently, `MultiHeadSelfAttention` is not exposed to the users. The `MultiHeadAttention` is more generic since it is not just self attention. We can try to make them similar without making it generic by only adding the args we used.